packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -99,50 +114,95 @@ class DefaultMaterialLocalizations implements MaterialLocalizations {
   /// have been translated.
   final Locale locale;
 
+  // TODO(hmuller): the rules for mapping from an integer value to
+  // "one" or "two" etc. are locale specific and an additional "few" category
+  // is needed. See http://cldr.unicode.org/index/cldr-spec/plural-rules
+  String _nameToPluralValue(int count, String key) {
+    String text;
+    if (count == 0)
+      text = _nameToValue['${key}Zero'];
+    else if (count == 1)
+      text = _nameToValue['${key}One'];
+    else if (count == 2)
+      text = _nameToValue['${key}Two'];
+    else if (count > 2)
+      text = _nameToValue['${key}Many'];
+    text ??= _nameToValue['${key}Other'];
+    assert(text != null);
+    return text;
```

Comment:
you're starting to reinvent package:intl here. I think we should consider whether it's time to extract all this into a separate package that just uses intl instead.

Reply:
I agree that a small package, based on package:intl, would be a good place for our number, date, plural (etc) needs.  Apps could choose to use our package.

Reply:
I think we'd be better off just using intl directly. We don't want to reinvent that wheel.